arrow-libs/optics/arrow-optics-ksp-plugin/src/test/kotlin/arrow/optics/plugin/LensTests.kt

```diff
@@ -38,7 +38,6 @@ class LensTests {
   fun `Lenses which mentions imported elements`() {
     """
       |$imports
-      |import kotlin.time.Duration
```

Comment:
These tests were not running if this bug was not caught. Is it possible to make them run?
If this import is removed, then it should be using FQN on line 47.

Reply:
Was removed since I added a variable in the dslModel with a Duration type so the import is automatically added in $import

Reply:
Let me change those to FQNames instead ☺️